src/ros_graph_parser/snapshot.py

```diff
@@ -9,7 +9,7 @@
 BLACK_LIST_PARAM = ['/rosdistro', '/rosversion', '/run_id']
 BLACK_LIST_TOPIC = ["/tf", "/tf_static", "/rosout", "/clock"]
 BLACK_LIST_SERV = ["/set_logger_level", "/get_loggers"]
-BLACK_LIST_NODE = ["/rosout","/head_cam"]
+BLACK_LIST_NODE = ["/rosout", "/head_cam"]
```

Comment:
ups! I introduced here an error! I will remove the "/head_cam" from the list

Reply:
Done https://github.com/ipa-nhg/ros_graph_parser/pull/5/commits/48b0618e75bbe2d0778296936e9a818f3aafd4be, @ipa-hsd you have to rebase/pull before you continue working on this file